ONSdigital/census-rm-toolbox
# 984 Add script for response driven reminders (LSOAs)

# Motivation and Context

RM will be given lists of LSOAs to generate response-driven reminders for. This PR adds 2 scripts:
* `reminder_lsoa_count.py` - takes a CSV file of LSOAs and returns a count of cases in scope for the reminder
* `reminder_lsoa.py` - takes a CSV file of LSOAs and creates an action rule based on the action type/pack code set

**Note**: It has been agreed that RM will not perform any validation on the LSOA files - the business area will be responsible for validating instead.

# What has changed

* Added script to ingest LSOAs file and return a count of cases in scope for reminder
* Added script to ingest LSOAs file and insert action rule into DB
* Renamed folders

# How to test?



* Build and deploy this pod (and the other branches from this ticket)
* Load HH sample file
* Load an LSOAs file (grab some LSOAs from your sample file) to the pod 
* Follow README instructions to get a count of the cases expected to be in the reminder
* Follow README instructions to generate an action rule
* Check that the print files have been generated as expected for the action type/pack code

# Links



https://trello.com/c/tRiFtYj5/984-reminders-response-driven-8
